fast-list.js

```diff
@@ -304,6 +303,9 @@ FastList.prototype = {
         item.dataset.detailPopulated = false;
         source.populateItem(item, i);
         item.dataset.populated = true;
+        if (item.style.display === 'none') {
+          item.style.removeProperty('display');
```

Comment:
Didn't know about this method! Any advantage to using `item.style.display = ''`?